core/new-gui/src/app/workspace/service/workflow-graph/model/workflow-action.service.ts

```diff
@@ -1075,7 +1075,7 @@ export class WorkflowActionService {
     this.toggleSendData(true);
   }
 
-  public workflowChanged(): Observable<void> {
+  public workflowChanged(): Observable<unknown> {
```

Comment:
the observable has a lot of possible types, are we encouraged to use `unknown` to describe them?

Reply:
yes, because we are not checking the actual value of the event, we just want to know that the event happens. 
In latest Typescript, `void` is no longer assignable to the union of all these types, so we use `unknown` instead. 

`unknown` is a type-safe counterpart of `any`. `any` is not good because the compiler allows you to do anything with the event value without any checks, `unknown` is good because the compiler **doesn't** allow you to do anything

Reply:
make sense, will use `unknown` for similar cases.